### DISCUSSION TITLE
Add mono flatpaks to flathub

### DISCUSSION THREAD

**Discussion by bbrainstormer** [godotengine/godot-proposals#7145] [Platforms]
Both godot 4 and godot 3 are available as flatpaks.  However, neither of these include c# support.  It would be convient to have godot with c# available directly from flathub, either in the base flatpak or as a seperate one.  I can't imagine it would be hard to do.

However, if I'm wrong, and there is some reason for this, I at least want to know why.

**Comment by Calinou** [comment]
This is already being tracked in https://github.com/flathub/org.godotengine.Godot/issues/8.

>  I can't imagine it would be hard to do.

Unfortunately, it's hard to do as building Mono (3.x) or .NET (4.x) from source is difficult. We also need to maintain separate manifests as not everyone needs C# support, and having to install C# support makes the Flatpak much larger.

**Reply by bbrainstormer** [reply]
Well, that sucks.  Thanks for the info.
